presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestPartitionTransforms.java

```diff
@@ -26,12 +26,12 @@
     @Test
     public void testToStringMatchesSpecification()
     {
-        assertEquals(Transforms.identity(StringType.get()).toString(), "identity");
-        assertEquals(Transforms.bucket(StringType.get(), 13).toString(), "bucket[13]");
-        assertEquals(Transforms.truncate(StringType.get(), 19).toString(), "truncate[19]");
-        assertEquals(Transforms.year(DateType.get()).toString(), "year");
-        assertEquals(Transforms.month(DateType.get()).toString(), "month");
-        assertEquals(Transforms.day(DateType.get()).toString(), "day");
-        assertEquals(Transforms.hour(TimestampType.withoutZone()).toString(), "hour");
+        assertEquals(Transforms.identity().toString(), "identity");
```

Comment:
specifying the type has been deprecated and will be removed in 2.0.0. The type is only needed at bind time, so we can use the new APIs here rather than waiting until this will be removed

Reply:
Nice!